.github/workflows/protos-up-to-date.yaml

```diff
@@ -0,0 +1,30 @@
+name: Protobufs up to date
+run-name: make protos-up-to-date
+on:
+  push:
+    branches:
+      - master
+
+  # These are required for the check to run on PRs.
+  pull_request:
+  merge_group:
+
+jobs:
+  protos-up-to-date:
+    if: ${{ !startsWith(github.head_ref, 'dependabot/') }}
+    name: Protobufs up to date
+    runs-on: ubuntu-22.04-16core
```

Comment:
We probably don't need a 16-core machine for this one.

Reply:
Ahhh yeah, silly copy paste. I'll reduce this.